influxql/parser.go

```diff
@@ -577,6 +577,15 @@ func (p *Parser) parseSelectStatement(tr targetRequirement) (*SelectStatement, e
 		return nil, err
 	}
 
+	// Set if the query is a raw data query or one with an aggregate
+	stmt.IsRawQuery = true
```

Comment:
Those should probably be properties that get set on parse. Like everything else here: https://github.com/influxdb/influxdb/blob/master/influxql/parser.go#L518-L580

And actually, `Aggregated` should be removed because it's representing the same thing as `IsRawQuery` except it's using different logic to do so. We'd be better off removing that and keeping things consistent.